ansible/configs/ibm-rhoic/destroy_env.yml

```diff
@@ -87,7 +87,7 @@
       uri:
         url: "{{ ibm_cloud_api_container_v1_url }}/rhpds-{{ guid }}?{{ ibm_cloud_api_rhoic_del_uri_parameters }}"
         method: DELETE
-        status_code: 204,404,-1,500
+        status_code: 204,404,500,502,-1
```

Comment:
Why are you looking for so many codes? We should only be waiting for a 204 here. That is the only thing that is success.

Reply:
because I am just throwing things at it to make it go, this has been cleaned up.